newsrevenuehub/rev-engine
# DEV-1806 | Address security concerns for JS dependencies

Please fill out each section even if it's just with "N/A"

#### Plan? (if this draft/incomplete indicate what you intend to do and how)

#### What's this PR do?

Fixes a "moderate" issue reported by `npm audit` with `terser` package. After running `npm audit fix` this issue is fixed, and `package-lock.json` is updated. 

Note that the known issue with `nth-check` remains, but we have determined that is not actionable at the moment, and does not apply to production dependencies.

#### Why are we doing this? How does it help us?

#### How should this be manually tested?

Pull down this branch, `cd ./spa`, and run `npm audit`. You should only see a message about `nth-check`.

#### Have automated unit tests been added?

N/A

#### How should this change be communicated to end users?

#### Are there any smells or added technical debt to note?

#### Has this been documented? If so, where?

#### What are the relevant tickets? Add a link to any relevant ones.

[DEV-1806](https://news-revenue-hub.atlassian.net/jira/software/c/projects/DEV/boards/3?modal=detail&selectedIssue=DEV-1806)

#### Do any changes need to be made before deployment to production (adding environment variables, for example)?

No

#### Are there next steps? If so, what? Have tickets been opened for them?

N/A
